src/style.css

```diff
@@ -0,0 +1,112 @@
+html {
```

Comment:
I'd recommend spreading styles over several files according to their dedication (resetting, normalizing, BEM-blocks)